rfcs/0010-postmortem-principles.md

```diff
@@ -0,0 +1,57 @@
+---
+created-on: 2021-04-27
+freshness: 2021-04-27
+owner: @maxneuvians
+status: proposal
+tags:
+  - postmortem
+  - incident response
+---
+
+[La version française suit.](#résumé)
+
+# CDS Principles for running incident response and doing postmortems
+
+## Summary
+Things go wrong. Software breaks. Partnerships aren’t smooth. Delivery takes longer than expected. Sometimes, things go right. In all cases, we should take the time to look back on what happened, how it happened and what we can do better in the future.
+
+Postmortems are a tool for determining what happened, documenting the timelines and discovering what went right and what went wrong. They help us understand how a given outcome occurred and, if necessary, provide the action items to ensure that outcome doesn’t happen again in the future.
+
+## Motivation
+We need to align as an organisation on the value of postmortems - what they give us, why they are worth investing in, and what the consequences of ignoring outcomes are.
+
+## Proposal
+
+### People will make mistakes
+It is completely unreasonable to expect humans to never make a mistake, otherwise they would never learn. When people make mistakes, we assume they acted with the best intentions based on the information they had at the time. A postmortem needs to honestly and objectively examine the circumstances that led to the fault so we can find the true root cause(s) and mitigate them.
+
+### People need to feel psychologically safe
```

Comment:
Again can we rephrase this as a proposal, maybe like: 

"We will create an environment of psychological safety."

Reply:
Thank you!